shared/common-adapters/user-proofs.js.flow

```diff
@@ -34,4 +34,4 @@ export type Props = ({
   missingProofs: Array<MissingProof>,
 } & CommonProps)
 
-export default class Render extends React$Component<void, Props, void> {}
+export default class ProofsRender extends React$Component<void, Props, void> {}
```

Comment:
I believe (with 85% confidence) it should be called `UserProofs` since that is the name of the component we import?